docs/content/references/config/index.md

```diff
@@ -0,0 +1,27 @@
+# Configuration
+
+HedgeDoc can be configured via environment variables either directly or via an `.env` file.
+
+## The `.env` file
+
+The `.env` file should be placed in the root of the project and contains key-value pairs of
+environment variables and their corresponding value. This can for example look like this:
+
+<!-- markdownlint-disable proper-names -->
+```ini
+HD_BASE_URL="http://localhost:8080"
+HD_SESSION_SECRET="change_me_in_production"
+HD_DATABASE_TYPE="sqlite"
+HD_DATABASE_NAME="./hedgedoc.sqlite"
+HD_MEDIA_BACKEND="filesystem"
+HD_MEDIA_BACKEND_FILESYSTEM_UPLOAD_PATH="uploads/"
+```
+<!-- markdownlint-enable proper-names -->
+
+We also provide an `.env.example` file containing a minimal configuration
+in the root of the project. This should help you to write your own configuration.
```

Comment:
Hmm, we now have the `.env.example` in the repo root, the inline example above and `files/setup-docker/config.env`.

That seems vulnerable to becoming inconsistent